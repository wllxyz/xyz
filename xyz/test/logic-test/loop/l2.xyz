<i>-->"\n"==><o>-->
	($PUSH_DATA)
	($SET, "n", "10")
	($LOOP, 
		($IGNORE, ($NOT, ($EQ, ($GET, "n"), "0"))),
		($IGNORE, 
				($SET, "s", "0")
				($SET, "i", "0")
			    	($LOOP, 
				    ($IGNORE, ($NOT, ($EQ, ($GET, "i"), ($GET, "n")))), 
				    ($IGNORE, 
					    ($SET, "s", ($ADD, ($GET, "s"), ($GET, "i")))
					    ($SET, "i", ($ADD, ($GET, "i"), "1"))
				    )
			    	)
			    	($GET, "s")
			    	"\n"
			    	($SET, "n", ($SUB, ($GET, "n"), "1"))
		)
	)
	($POP_DATA);
