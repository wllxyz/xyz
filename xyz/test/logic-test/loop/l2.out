45
36
28
21
15
10
6
3
1
0
